quiz.js

```diff
@@ -1,25 +1,31 @@
-var tree = {};
+
 var growButton = document.getElementById('grow_button');
-growButton.addEventListener("click", runner);
+growButton.addEventListener("click", function () {
+  var tree = {};
+  tree.height = treeHeight.value;
+  tree.branch = branchChar.value;
+  runner(tree)
+});
 var showTree = document.getElementById('show_tree');
 var branchChar = document.getElementById('branch_character');
 var treeHeight = document.getElementById('tree_height');
-branchChar.addEventListener("keyup", enterF);
+document.body.addEventListener("keyup", enterF);
 
 function enterF (e) {
   if (e.key === 'Enter') {
+    var tree = {};
```

Comment:
Lines 16, 17 & 18 are the same as lines 4, 5, & 6. How could you make this code DRYer?